README.md

```diff
@@ -2,18 +2,11 @@
 
 # Project Name
 
-> Description the project.
-
+This is a small project to practice linters
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+html, and css
 
 
 ## Getting Started
```

Comment:
- [ ] Kindly customize the getting start section by adding some steps to how the user can work with the project.
   e.g 
  ```
   git clone repo.git
   cd repo
   ```